### ISSUE TITLE
Refactor 

### ISSUE THREAD

**Issue by kalabiyau** [openSUSE/osem#597]
https://github.com/openSUSE/osem/blob/master/app/controllers/proposal_controller.rb#L24

ProposalController#create to be more simple and understandable. 
Tests should stay green, code should be simplified

**Comment by AdiChat** [comment]
Could be elaborate on which part of the create method to simplify or what is your vision? I went through the code and it seemed understandable.

**Comment by chocoelho** [comment]
I'll work on it :+1:

**Comment by Shriyanshagro** [comment]
I have gone through the code and it seems quite interesting
So is this issue still open ??

**Comment by sonalkr132** [comment]
@Shriyanshagro yes, this issue is open. Take a look at comments on #822 before you start working on it.

**Comment by hennevogel** [comment]
I'm pretty sure that there is nothing to re-factor here. All this method needs is some inline comments. Not a Junior job
